src/script/script_error.h

```diff
@@ -59,12 +59,15 @@ typedef enum ScriptError_t
     SCRIPT_ERR_MINIMALDATA,
     SCRIPT_ERR_SIG_PUSHONLY,
     SCRIPT_ERR_SIG_HIGH_S,
-    SCRIPT_ERR_SIG_NULLDUMMY,
+    SCRIPT_ERR_MULTISIG_HINT,
```

Comment:
`HINT` sounds a bit vague. Why not just `SIGMAP` or `SKIPPED_PUBKEYS` or something?

Reply:
`SCRIPT_ERR_MULTISIG_SKIPPED_PUBKEYS` it is.